echemdb/cv/cv_database.py

```diff
@@ -0,0 +1,110 @@
+r"""
+A Database of Cyclic Voltammograms.
```

Comment:
```suggestion
A Database of Cyclic Voltammograms
```

I think these show up as `.` in the heading of the documentation (which does not look right to me.)

Reply:
The title is actually the name of the module and this is the text appearing below the title. Thus I would keep it as is.